### ISSUE TITLE
subzero cache files causing hangs, crashes and Windows Backup to Fail.

### ISSUE THREAD

**Issue by nydave69** [pannal/Sub-Zero.bundle#600]
Spent the last 3 weeks trying to figure out whats happening, the results..

files in the cache directory have random numbers for the max length of the file name.
Windows 7 in this case has issues with file name that long.
any access to those files create a hang or crash of that process.
Can't do s system backup, it crashes it, the error message from backup is a catchall..
can't open the files or move them or delete them using the GUI.. File to long error message.
windows directory is;
C:\Users\Plex\AppData\Local\Plex Media Server\Plug-in Support\Data\com.plexapp.agents.subzero\cache

![subzero bug](https://user-images.githubusercontent.com/14140816/48980076-72fee900-f092-11e8-9de5-a7b3a0e8bdfc.PNG)

only thing I could do is delete them in a command window...

Version file in the bundle: e3034be (Mon Nov 19 16:17:11 UTC 2018)

**Comment by pannal** [comment]
Interesting. I wasn't aware of Windows still having the 260 char limit in 2018. Apparently this can be [removed](https://www.howtogeek.com/266621/how-to-make-windows-10-accept-file-paths-over-260-characters/).

For now you can disable `Use new style caching (for subliminal)` in SZ's settings to circumvent this.

I'll fix this in the next version.

**Comment by pannal** [comment]
It's on by default as it works flawlessly now. I wasn't aware of the implication on Windows, though :)

**Comment by pannal** [comment]
This is fixed in current develop-2.6. Care to try?

**Comment by nydave69** [comment]
well I've unchecked it and left it alone for a few days now.
I'm still seeing a problem, it looks like the same type of files but I've checked the ones in the cache folder and it's not in there. Only way I can fine them again will be to cut and paste the whole directory skipping the ones that are a problem.  When I have more time and less users on.

**Comment by nydave69** [comment]
Just a thought... I backup to a Synology NAS and the number of allowed characters is less then Windows..  
Will see if that plays into it.

**Comment by nydave69** [comment]
Just an update;
Yes I did clean the cache folder after changing the setting and restarted everything...

It seems that there was another file that was causing the issue, not in the cache folder. 

I copied the whole Plex Plugins folder to another drive and skipped the error files.
Deleted the original Plex Plugins folder location and copied everything back.

Has been running since (about 4 days) without issues.
